data/org.freedesktop.portal.Flatpak.xml

```diff
@@ -265,7 +265,7 @@
                <filename>/run/parent/usr</filename>, with filenames like
                <filename>/run/parent/usr/bin/env</filename>,
                and compatibility symlinks like
-               <filename>/run/parent/bin</filename> →
+               <filename>/run/parent/bin</filename> ->
```

Comment:
If we really can't use Unicode,

```suggestion
               <filename>/run/parent/bin</filename> -&gt;
```

or

```suggestion
               <filename>/run/parent/bin</filename> &#x2192;
```

Reply:
I think we need one or the other of my suggestions here. Sorry, I've tried and failed to find a way to make Python 3.5 more Unicode-friendly.